content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -232,4 +306,15 @@ Registering the O3DE engine enables O3DE projects to find the engine, even when
 
     The O3DE manifest file is `<USER_DIRECTORY>/.o3de/o3de_manifest.json`. The paths to all the registered engines, projects, and more are recorded in this file.
 
+{{% /tab %}}
+{{< /tabs >}}
+
 You are now ready to create a project! For an introduction to project configuration, refer to [Project Creation with Open 3D Engine](/docs/welcome-guide/create).
+
+## Side-by-side engines
+ 
+ In the preceding sections, you learned how to set up and configure the O3DE engine as either a source engine or a pre-built SDK engine. You also have the option of registering both engines in a side-by-side configuration. This is one way to isolate your engine source code development from your project development.
+
+ For example, if the engine source is in `C:\o3de` and the pre-built SDK engine is in `C:\o3de-install`, you can give each engine its own engine name so that you can register both in the O3DE manifest using the `o3de` script's `register` command. To test your engine modifications, you can build a test project using the source engine project creation instructions, which builds the engine in the project directory. When you're ready to update the SDK engine used by your production project, you can build a new version of the SDK engine from `C:\o3de` using the `INSTALL` target. This will update the binaries in `C:\o3de-install`.
```

Comment:
```suggestion
 For example, if the engine source is in `C:\o3de` and the pre-built SDK engine is in `C:\o3de-install`, you can give each engine its own engine name so that you can register both in the O3DE manifest using the `o3de` script's `register` command. To test your engine modifications, you can build a test project using the source engine project creation instructions, which builds the engine in the project directory. When you're ready to update the SDK engine that your production project uses, you can build a new version of the SDK engine from `C:\o3de` using the `INSTALL` target. This updates the binaries in `C:\o3de-install`.
```